### PR TITLE
CCD-1608 : Fortify Scan implementation for aac-manage-case-assignment

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-1608

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!groovy
+@Library("Infrastructure")
+
+def type = "java"
+def product = "ccd"
+def component = "spring-boot-template"
+
+withNightlyPipeline(type, product, component) {
+  enableFortifyScan('ccd-aat')
```
Comment (kaanaktas) [inline Jenkinsfile_nightly:9]: no need parameter, as product is already ccd.

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!groovy
+@Library("Infrastructure")
+
+def type = "java"
+def product = "ccd"
```
Comment (MSancaktutar) [inline Jenkinsfile_nightly:5]: product is actually aac.

File: Jenkinsfile_nightly (hunk 1 above)
Comment (MSancaktutar) [reply Jenkinsfile_nightly:9]: product is actually aac.

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**